unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/ProxySettingsRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace DCL.SettingsCommon
+{
+    public class ProxySettingsRepository<T> : ISettingsRepository<T> where T : struct
+    {
+        private readonly ISettingsRepository<T> latestRepository;
+        private readonly ISettingsRepository<T> fallbackRepository;
```

Comment:
It took me a bit to find the usage of this class. 

When I read `fallback` I assume that this repository tries to write in the `latest` one and if fails goes to the `fallback` when, actually, the class is doing something different, It just uses the `fallback`to fill the data.

Maybe something like `backupRepository` would be more clear?

Reply:
Maybe `defaultSettingsRepository` to make the intent very explicit?

Reply:
I dont think it's being used as a default one, is to transition the current implementation to the new one

Reply:
i think its being used as a data recovery, perhaps `recoveryRepository`? or maybe just `oldRepository`? i dont think that `backup` or `default` fits the purpose of the object